src/commercetools/testing/abstract.py

```diff
@@ -277,6 +287,18 @@ def delete_by_key(self, request, key):
             return create_commercetools_response(request, json=obj)
         return create_commercetools_response(request, status_code=404)
 
+    def delete_by_container_key(self, request, container, key):
```

Comment:
Maybe we should name it `delete_by_container_and_key` to avoid users misreading it as deleting a bunch of objects by container key. But this depends on whether this naming convention is used elsewhere in a similar way.